### PR TITLE
chore: adding circle-ci

### PR DESCRIPTION
**Pull request by aleku399**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:9
```
Comment (epicallan) [inline .circleci/config.yml]: why not version 10

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: yarn install
```
Comment (epicallan) [inline .circleci/config.yml]: let's use `npm`, this means changing all the other `yarn` commands

Comment (aleku399) [reply .circleci/config.yml]: ok but i could prefer we use yarn for everything

Comment (epicallan) [reply .circleci/config.yml]: My experience with yarn is that at times yarn.lock files get corrupted and you have to delete yarn.lock and at times it's not so obvious. You can leave it in CI if you are still convinced you to prefer it to `npm`.

Comment (aleku399) [reply .circleci/config.yml]: ok am i good to merge

Comment (epicallan) [reply .circleci/config.yml]: yes

**Pull request merged**